### ISSUE TITLE
Fresh Install Doesn't Work Out of the Box?

### ISSUE THREAD

**Issue by Petercopter** [rspec/rspec-rails#1848]
I've been using RSpec for a long time, and now I am upgrading to Rails 5, so I figured I'd run a fresh install of rspec-rails, and see what configuration changes there may be.

After the fresh install, I tried running the test suite, but all I'm getting is `Uninitialized Constant`.

I'm looking at the `.rspec` file, and it's got `--require spec_helper` in it. So I think that means it's including the `spec_helper.rb` file in all specs. But that file doesn't include the `rails_helper` file, it's actually the other way around. `rails_helper` is including `spec_helper`.

If I change `.rspec` to `--require rails_helper`, I seem to be getting further.

So I looked at the docs, and there is
> NOTE: Generators run in RSpec 3.x will now require rails_helper instead of spec_helper.

But that doesn't seem to be the case. Would that be the include line at the top of the spec files? I removed those, in favor of the `.rspec` method, which is the same thing in the fresh install.

Am I doing something wrong? Is this something that needs to be corrected? I'm pretty sure I can work around it, but this could be confusing for someone new to Rspec.

Thanks so much for the hard work on this gem, I really appreciate it!

**Comment by tvanderpol** [comment]
I tried to replicate this. 

I used Rails 5.2.1 and added `gem 'rspec-rails'` to the `group :development, :test` in my Gemfile

I ran `bundle install` and `rails generate rspec:install`. The `bundle exec rspec` command works without issues.

To make sure I didn't miss something I generated a model (this worked properly, generating specs and all) and after a db migration the specs also ran without trouble (that is, the stub generated, nothing was actually tested in a fresh Rails install).

Without more info I can't work out what the problem here is/was.
